### DISCUSSION TITLE
How to get the partitions after using --compute-partitions and append to the original problem?

### DISCUSSION THREAD

**Discussion by yflich** [cvc5/cvc5#11220] [Q&A] [answered]
I 'm trying to use --compute-partitions to split the original problem, and then append the partition to the original problem if the result is unsat.
cvc5 will dump the partitions to stdout, or to the file specified by the option --write-partitions-to , but is there any way from C++ api to get the partitions?

Thanks

**Comment by amaleewilson** [comment, marked as answer]
> I 'm trying to use --compute-partitions to split the original problem, and then append the partition to the original problem if the result is unsat.
> 
> cvc5 will dump the partitions to stdout, or to the file specified by the option --write-partitions-to , but is there any way from C++ api to get the partitions?
> 
> 
> 
> Thanks

I'm working on adding this to main.

**Reply by yflich** [reply]
Thanks
